Sources/Datadog/RUM/RUMTelemetry.swift

```diff
@@ -48,7 +49,8 @@ internal final class RUMTelemetry: Telemetry {
         dateProvider: DateProvider,
         configurationEventMapper: RUMTelemetryConfiguratoinMapper?,
         delayedDispatcher: RUMTelemetryDelayedDispatcher?,
-        sampler: Sampler
+        sampler: Sampler,
+        configurationExtraSampler: Sampler = Sampler(samplingRate: 0.2) // Extra sample of 20% for configuration events
```

Comment:
I'm not sure if this is the most Swift idiomatic way.
Happy to take feedback.

Reply:
It is 👍